package.json

```
@@ -44,6 +45,13 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org"
   },
+  "lint-staged": {
+    "*.json": "prettier --write",
+    "*.ts": [
+      "prettier --write",
+      "eslint --fix"
+    ]
```

I think we should probably do this (since we run `prettier --check .`, which is basically "prettier all the things!!"):

```suggestion
    "*": "prettier --ignore-unknown --write",
    "*.ts": "eslint --fix"
```

Do you know if that syntax works, specifically does the `*.ts` and `*` both run for typescript files?

---

thanks, I added that.